src/plugins/intel_cpu/tests/functional/single_layer_tests/nms_rotated.cpp

```diff
@@ -0,0 +1,338 @@
+                ::testing::Values(0.3f, 0.7f),              // Score threshold
+                ::testing::Values(true, false),             // Sort result descending
+                ::testing::Values(true, false),             // Clockwise
+                ::testing::Values(false),                   // Is 1st input constant
+                ::testing::Values(false),                   // Is 2nd input constant
+                ::testing::Values(false),                   // Is 3rd input constant
+                ::testing::Values(false),                   // Is 4th input constant
+                ::testing::Values(false),                   // Is 5th input constant
+                ::testing::Values(emptyCPUSpec),            // CPU specific params
+                ::testing::Values(empty_plugin_config)),    // Additional plugin configuration
+        NmsRotatedLayerTestCPU::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(nightly_Static, NmsRotatedLayerTestCPU,
+        ::testing::Combine(
+                ::testing::Values(input_shapes[0], input_shapes[1], input_shapes[2], input_shapes[3]),
+                ::testing::Values(ElementType::f16, ElementType::bf16),
+                ::testing::Values(ElementType::i64),
+                ::testing::Values(ElementType::f16, ElementType::bf16),
+                ::testing::Values(ElementType::i64),
+                ::testing::Values(10),
+                ::testing::Values(0.5f),
+                ::testing::Values(0.4f),
+                ::testing::Values(true, false),
+                ::testing::Values(true, false),
+                ::testing::Values(true, false),
+                ::testing::Values(true, false),
+                ::testing::Values(true),
+                ::testing::Values(true),
+                ::testing::Values(true),
+                ::testing::Values(emptyCPUSpec),
+                ::testing::Values(empty_plugin_config)),
+        NmsRotatedLayerTestCPU::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(nightly_Dynamic, NmsRotatedLayerTestCPU,
+        ::testing::Combine(
+                ::testing::Values(input_shapes_nightly_dyn),
+                ::testing::Values(ElementType::f16, ElementType::bf16),
+                ::testing::Values(ElementType::i64),
+                ::testing::Values(ElementType::f16, ElementType::bf16),
+                ::testing::Values(ElementType::i64),
+                ::testing::Values(10),
+                ::testing::Values(0.5f),
+                ::testing::Values(0.4f),
+                ::testing::Values(true, false),
+                ::testing::Values(true, false),
+                ::testing::Values(true, false),
+                ::testing::Values(true, false),
+                ::testing::Values(true),
+                ::testing::Values(true),
+                ::testing::Values(true),
+                ::testing::Values(emptyCPUSpec),
+                ::testing::Values(empty_plugin_config)),
+        NmsRotatedLayerTestCPU::getTestCaseName);
```

Comment:
Is there any reason why we separate static and dynamic shapes for nightly scope?

Reply:
Initially it had. United now